zcash_transparent/src/zip48.rs

```diff
@@ -229,6 +257,18 @@ impl FullViewingKey {
         }
     }
 
+    /// Returns the ZIP 48 coin type and account ID for this full viewing key.
+    fn coin_type_and_account(&self) -> (u32, AccountId) {
```

Comment:
nit: do we not have a `CoinType` type anywhere? If not, we should add one at some point.